### ISSUE TITLE
New google closure compiler jar file naming

### ISSUE THREAD

**Issue by nomean42** [dcodeIO/ClosureCompiler.js#44]
Due to release notes July 13, 2016 [there](https://github.com/google/closure-compiler/wiki/Releases) is new compiler jar file naming system.  There should be added something like 
```javascript
//line 166
 var compilerFile = '';
        fs.readdirSync(__dirname + '/compiler').forEach(function(file) {
            if (file.indexOf('closure-compiler') !== -1) {
                compilerFile = file;
            }
        })
        var args = ['-XX:+TieredCompilation'];
            if (xms) args.push('-Xms'+xms);
            args.push(
                '-Xmx'+xmx,
                '-jar', __dirname+'/compiler/' + compilerFile
            );
```

into ClosureCompiler.js

**Comment by screenpunch** [comment]
Was just about to raise this myself.

protobuf.js is failing `compile ` with

`Error: Unable to access jarfile [SOMEPATH]/protobuf.js/node_modules/closurecompiler/compiler/compiler.jar`

**Comment by alex88** [comment]
Is there a workaround in the meantime?

**Comment by dickie81** [comment]
Created pull request that contains fix the issue:

https://github.com/dcodeIO/ClosureCompiler.js/pull/45

**Comment by dcodeIO** [comment]
Available on npm as version 1.6.0.

**Comment by dickie81** [comment]
Thanks!
